holopy/core/io/vis.py

```diff
@@ -261,7 +261,9 @@ def show_scatterer_slices(scatterer, spacing):
     show2d(vol)
 
 def test_display():
-    #diagnostic test to check matplotlib backend.
+    """Diagnostic test to check matplotlib backend.
+
+    You should see a white square inside a black square, with a colorbar."""
```

Comment:
You should also be able to use arrow keys to switch the square into a diamond and a circle. This should be added to the description, since that is the main purpose of this function.

Reply:
Should be fixed by 072c2e1